t/plugin/udp-logger.t

```diff
@@ -425,7 +425,6 @@ qr/.*plugin_metadata.*/
             end
 
             ngx.say(body)
-
```

Comment:
The `ngx.say` has nothing to do with the code below, so the blank is needed?

Reply:
Yes but I am unsure. I was getting lint error previously so I added this new line.